crates/runtime/src/pallets.rs

```diff
@@ -151,12 +183,17 @@ impl pallet_timestamp::Config for Runtime {
     type WeightInfo = ();
 }
 
+fn get_execution_constants() -> Arc<VersionedConstants> {
+    EXECUTION_CONSTANTS.deref().clone()
```

Comment:
do we need to clone?

Reply:
yep, otherwise it's the ref and doesn't work